server/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -874,6 +874,9 @@ public IndexMetaData verifyIndexIsDeleted(final Index index, final ClusterState
             final IndexMetaData metaData;
             try {
                 metaData = metaStateService.loadIndexState(index);
+                if (metaData == null) {
+                    throw new IllegalStateException("Index state does not exist.");
```

Comment:
I wonder if we should just ignore it instead of throwing exception here and therefore logging a warning.

Reply:
makes sense removing the warning :)

Reply:
done